runelite-client/src/test/java/net/runelite/client/plugins/itemstats/ItemStatEffectTest.java

```diff
@@ -94,13 +147,38 @@ public void prepare()
 			.thenReturn(0);
 	}
 
+	private void prepareInventory()
```

Comment:
This looks like it's left behind from the holy wrench testing that was removed.

Reply:
One of these force pushes will be right i promise.